src/module.c

```diff
@@ -434,8 +434,6 @@ int DeleteCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
   RedisModule_AutoMemory(ctx);
 
   if (argc < 3 || argc > 4) return RedisModule_WrongArity(ctx);
-  RedisModule_ReplicateVerbatim(ctx);
```

Comment:
Why not using RedisModule_ReplicateVerbatim?

Reply:
For many successive commands, this saves the need for the replicas to look up the alias.